src/exampleTypes/ResultPager.js

```diff
@@ -0,0 +1,69 @@
+import _ from 'lodash/fp'
+import React from 'react'
+import { observer } from 'mobx-react'
+import InjectTreeNode from '../utils/injectTreeNode'
+
+// These are to prevent warning from `active`, `previous`, `next`
+let span = ({ children }) => <span>{children}</span>
+let a = ({ children, onClick }) => <a onClick={onClick}>{children}</a>
+
+export default InjectTreeNode(
+  observer(({ node, tree, List = 'div', Item = span, Link = a }) => {
+    let pages = Math.ceil(
+      (node.context.response.totalRecords || 1) / node.pageSize
```

Comment:
Is it safe to assume that `node.context.response.totalRecords` exists? also, would it make sense to have a default value for pageSize?

Reply:
Both are provided by contexture-client. That's a great example on why I want to pull example types out to their own repo instead of spread out across all the different packages.